docs/source/essentials/get-started-multiplatform.mdx

```diff
@@ -36,6 +36,17 @@ kotlin {
 }
 ```
 
+## Multithreaded coroutines
+If using the x.y.z-native-mt branch of coroutines, gradle will replace the -native-mt version with the non-mt version as outlined [here](https://kotlinlang.org/docs/mobile/concurrency-and-coroutines.html#multithreaded-coroutines). To prevent this happening use the following:
+
+```
+implementation ("org.jetbrains.kotlinx:kotlinx-coroutines-core:x.y.z-native-mt"){
+    isForce = true
+}
+// Must come after the coroutines dependency
```

Comment:
I wouldn't expect the order of the declaration to be important. What happens if you swap the lines?

Reply:
You're right! I was misunderstanding the gradle docs on isForce